app/javascript/vue/repository_item_sidebar/customColumns.vue

```diff
@@ -15,6 +15,7 @@
         :updatePath="updatePath"
         :optionsPath="column.options_path"
         :inArchivedRepositoryRow="inArchivedRepositoryRow"
+        :canEdit="permissions?.can_manage && !inArchivedRepositoryRow"
```

Comment:
Is there an option for permissions to be blank that is not just the consequence of loading? If not, it should probably go up in the if condition, as it is required to properly render the component:
`<div v-if="permissions && customColumns?.length > 0"`